libs/hybris/src/hooks_shm.c

```diff
@@ -94,9 +94,11 @@ static void _sync_mmap_with_shm()
             /* Note that mremap may change the address pointed by _hybris_shm_data.
              * But as we never point directly into _hybris_shm_data, it's fine.
              * */
-            _hybris_shm_data = (hybris_shm_data_t *)mremap( _hybris_shm_data, _current_mapped_size,
+             #ifndef __APPLE__
```

Comment:
Don't align the ifdef with the code. Ditto elsewhere

Reply:
Well, I wasn't entirely sure because the `#ifdef`s were a little inconsistent (headers [had it lined up](https://github.com/MCMrARM/mcpelauncher-linux/blob/eb96b8d35d1a358609f22a2f425a60fdc715b1d2/src/client/main.cpp#L51), code blocks had it tabbed back [sometimes](https://github.com/MCMrARM/mcpelauncher-linux/blob/eb96b8d35d1a358609f22a2f425a60fdc715b1d2/src/client/main.cpp#L391-L394))

I'll revisit that, though

Reply:
Yeah do not indent the ifdefs.